copy_assets.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Define source and destination directories
+srcDir="src"
+destDir="built"
+
+# Function to copy .csv and .json files
+copyFiltered() {
+  local srcDir="$1"
+  local destDir="$2"
+
+  # Create destination directory if it does not exist
+  mkdir -p "$destDir"
+
+  # Read all files and directories within current directory
+  local filesAndDirs=("$srcDir"/*)
+
+  for path in "${filesAndDirs[@]}"; do
```

Comment:
I guess this can this be replaced by `find ... -exec ...` with less lines of code and improved readability?